docs/error-messages/compiler-warnings/compiler-warning-level-1-c4172.md

```diff
@@ -26,7 +26,7 @@ The following example generates C4172:
 const int* func1()
 {
     int i = 42;
-	return &i; // C4172
+    return &i;   // C4172
```

Comment:
We should standardize use of spaces over tabs and the 3 spaces before `// CXXXX`.